server.js

```diff
@@ -25,8 +25,7 @@ const port = process.env.PORT || 5000;
 /* Import app after setting global.verbose so that calls to utils.verbose()
  * respect our --verbose option as expected.
  */
-/* eslint-disable import/first */
-import app from './src/app.js';
```

Comment:
> Node.js starts locating and executing static imports before the actual import statements are reached in the importing module's execution.

Is this analogous to normal JS hoisting?

Reply:
Kind of, as specific behavior – in ESM all static `import`s must be top-level and are analyzed/linked before everything is evaluated¹.

¹ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#description